### PR TITLE
teacher technology stack

### PR DESCRIPTION
**Pull request by professor-Ibisi**

### CODE REVIEW

File: myteacher.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        <h1>My teacher's technology stack</h1>
+        <ol>
+            <li>Ruby</li>
+            <li>Ruby on rails</li>
+            <li>Linux</li>
+            <li>JavaScript-Nodejs</li>
+            <li>Reactjs</li>
+        </ol>
+
+    </body>
```
Comment (leonardk1) [inline myteacher.html]: you have not closed the html tag
```
<html>
</html>
```

so add the closing tag, then push the work again

Comment (leonardk1) [reply myteacher.html]: also rename the file to be more meaningful it is talking about technology stack

so maybe the name should be `teacher-technology-stack.html`

`git mv myteacher.html teacher-technology-stack.html`

Comment (professor-Ibisi) [reply myteacher.html]: ok let me do

File: teacher-technology-stack.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        <h1>My teacher's technology stack</h1>
+        <ol>
+            <li>Ruby</li>
+            <li>Ruby on rails</li>
+            <li>Linux</li>
+            <li>JavaScript-Nodejs</li>
+            <li>Reactjs</li>
+        </ol>
+    </body>
+    </html>
```
Comment (leonardk1) [inline teacher-technology-stack.html]: Now kuna kitu kinaitwa indentation. ili code zako ziwe katika good format **(CODE READABILITY)**

```
<!DOCTYPE html>
<html>
  <head>
    <title>My teacher's technology</title>     
  </head>
  <body>
    <h1>My teacher's technology stack</h1>
    <ol>
      <li>Ruby</li>
      <li>Ruby on rails</li>
      <li>Linux</li>
      <li>JavaScript-Nodejs</li>
      <li>Reactjs</li>
    </ol>
  </body>
</html> 
```

https://courses.cs.washington.edu/courses/cse154/17au/styleguide/html-css/spacing-indentation-html.html
https://stackoverflow.com/questions/34174207/how-to-change-indentation-in-visual-studio-code/45671704

When working with me @leonardk1 , I use two space indentation. learn what `two space` indentation is. it is good for you to make you code look good, and readable by others

Comment (leonardk1) [reply teacher-technology-stack.html]: Check out these examples of how I use two space indentatation
```
def get_base_url(env, subdomain, api_path)
  url = case env
  when 'production'
    "https://#{subdomain}.my-portal.com/"
  when 'staging'
    "https://#{subdomain}.dev.home.com/"
  else
    "http://#{subdomain}.lvh.me:3000/"
  end

  url + api_path
end

def users(url)
  url = url + 'users'

  RestClient::Request.execute(method: :get,
                              url: url,
                              payload: {},
                              headers:{Authorization: "Bearer #{@token}"} )
end
```

Comment (leonardk1) [reply teacher-technology-stack.html]: also you have not yet renamed the file, to reflect its purpose. the naming of our files also has to be meaningful

Comment (professor-Ibisi) [reply teacher-technology-stack.html]: true

**Pull request merged**